clients/java/signalr/src/main/java/ConsoleLogger.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+public class ConsoleLogger implements Logger {
+    private LogLevel logLevel;
+    public ConsoleLogger(LogLevel logLevel) {
+            this.logLevel = logLevel;
+    }
+
+    @Override
+    public void log(LogLevel logLevel, String message) {
+        if(logLevel.value >= this.logLevel.value){
+            switch (logLevel) {
+                case Debug:
+                case Information:
+                    System.out.println(message);
+                    break;
+                case Warning:
+                case Error:
+                case Critical:
+                    System.err.println(message);
+                    break;
+            }
+        }
+    }
+
+    @Override
+    public void log(LogLevel logLevel, String formattedMessage, Object... args) {
+        if (logLevel.value >= this.logLevel.value) {
+            switch (logLevel) {
+                case Debug:
+                case Information:
+                    System.out.printf(formattedMessage, args);
```

Comment:
Does this print a newline at the end? I don't think it does from the [docs](https://docs.oracle.com/javase/7/docs/api/java/io/PrintStream.html). I think `System.out.println(String.format(formattedMessage, args))` is what we want here (and the same below for `System.err`)

Reply:
No new line in `printf` which is why I added them in the messages. Makes more sense to add it to the console logger 👍